### ISSUE TITLE
Add/Update [AKS Disk Encryption Set ID Undefined] query for [Terraform]

### ISSUE THREAD

**Issue by Carus11** [Checkmarx/kics#6022]
### Platform
Terraform

### Provider
Azure

### Description
The Query:
```json
"query_name": "AKS Disk Encryption Set ID Undefined",
"query_id": "b17d8bb8-4c08-4785-867e-cb9e62a622aa",
"query_url": "https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/kubernetes_cluster#disk_encryption_set_id",
"severity": "MEDIUM",
"platform": "Terraform",
"cloud_provider": "AZURE",
"category": "Encryption",
"description": "Azure Container Service (AKS) should use Disk Encryption Set ID",
```

Suggests that I use a Disk Encryption Set when I set up my AKS Cluster. If I attempt to do that in addition to using Ephemeral OS Disks, Azure gives me the following error:

```sh

│ Error: creating Managed Cluster (Subscription: "....."
│ Resource Group Name: "my-rg"
│ Resource Name: "my-dev-cluster"): managedclusters.ManagedClustersClient#CreateOrUpdate: Failure sending request: StatusCode=0 -- Original Error: Code="EphemeralOSAndBYOKNotSupported" Message="BYOK and Ephemeral OS disk can not be combined in Azure. Disable ephemeral OS disk or disable BYOK."
│
│   with azurerm_kubernetes_cluster.my_dev_cluster,
│   on k8s.tf line 1, in resource "azurerm_kubernetes_cluster" "my_dev_cluster":
│    1: resource "azurerm_kubernetes_cluster" "my_dev_cluster" {
```

Can I suggest amending this query, so that it does not warn me about disk encryption sets when I am choosing to use Ephemeral OS disks? Is that typically an acceptable security posture?

**Comment by cxMiguelSilva** [comment]
Hi @Carus11, hope you are doing great!
Thank you so much for reporting this bug.
A fix is already in #6023. Let me know what you think.

**Comment by Carus11** [comment]
It looks good!
Just don't presume I am an expert on best practices for disk encryption, I lean heavily on what kics tells me I should be doing!
